content/docs/snapshots/volume-group-snapshots/_index.md

```diff
@@ -9,12 +9,55 @@ Description: >
 The Dell CSM Volume Group Snapshotter is an operator which extends Kubernetes API to support crash-consistent snapshots of groups of volumes.
 Volume Group Snapshot supports PowerFlex and PowerStore driver.
 
-In order to use Volume Group Snapshots, ensure the volume snapshot module is enabled.
-- Kubernetes Volume Snapshot CRDs
-- Volume Snapshot Controller
-- Volume Snapshot Class
+## Installation
+To install and use the Volume Group Snapshotter, you need to install pre-requisites in your cluster, then install the CRD in your cluster and deploy it with the driver.
 
-### Creating Volume Group Snapshots
+### 1. Install Pre-Requisites
+The only pre-requisite required is the external-snapshotter, which is available [here](https://github.com/kubernetes-csi/external-snapshotter/tree/v4.1.1). Version 4.1+ is required. This is also required for the driver, so if the driver has already been installed, this pre-requisite should already be fulfilled as well.
+
+The external-snapshotter is split into two controllers, the common snapshot controller and a CSI external-snapshotter sidecar. The common snapshot controller must be installed only once per cluster.
+
+Here are sample instructions on installing the external-snapshotter CRDs:
+```
+git clone https://github.com/kubernetes-csi/external-snapshotter/
+cd ./external-snapshotter
+git checkout release-<your-version>
+kubectl create -f client/config/crd
+kubectl create -f deploy/kubernetes/snapshot-controller
+```
+
+### 2. Install VGS CRD
+
+```
+IMPORTANT: delete previous v1aplha2 version of CRD and vgs resources created using alpha version.
+	   Snapshots on array will remain if memberReclaimPolicy=retain was used.
+```
+If you want to install the VGS CRD from a pre-generated yaml, you can do so with the following command (run in top-level folder):
+```
+kubectl apply -f config/crd/vgs-install.yaml
+```
+
+If you want to create your own CRD for installation with Kustomize, then the command `make install` can be used to create and install the Custom Resource Definitions in your Kubernetes cluster.
+
+### 3. Deploy VGS in CSI Driver with Helm Chart Parameters
+The drivers that support Helm chart deployment allow the CSM Volume Group Snapshotter to be _optionally_ deployed 
+by variables in the chart. There is a _vgsnapshotter_ block specified in the _values.yaml_ file of the chart that will look similar the text below by default:
```

Comment:
Replace "the text below by default" to "this default text"

Reply:
Done